lte/gateway/c/core/oai/lib/openflow/controller/GTPApplication.cpp

```diff
@@ -171,7 +171,7 @@ void GTPApplication::add_uplink_tunnel_flow(
   apply_ul_inst.add_action(set_eth_dst);
 
   of13::SetFieldAction set_tunnel_id(
-      new of13::NXMRegX(TUNNEL_ID_REG, ev.get_in_tei()));
+      new of13::NXMRegX(TUNNEL_ID_REG, ev.get_out_tei()));
```

Comment:
Mm do we use this anywhere rn?

Reply:
nope, plan is to use it in ipv6 solicitation handler app.